pkg/events/parsers/data_parsers.go

```diff
@@ -1466,7 +1474,7 @@ func ParseMmapProt(rawValue uint64) MmapProtArgument {
 
 	var sb strings.Builder
 	for _, a := range mmapProtArgs {
-		if OptionAreContainedInArgument(rawValue, a) {
+		if optionIsContainedInArgument(rawValue, a.Value()) {
```

Comment:
`ParseMmapProt` already iterates its arguments. In the future, the loop approach can be applied to the other parser functions. Despite the improvement in CPU time the only criticism is that we need a global variable containing all arguments. But I believe this can be improved further by using pointers to existing declarations.